scripts/deploy.sh

```diff
@@ -33,12 +33,12 @@ function upload_assets {
   mv -f "${TLDR_LANG_ARCHIVES_DIRECTORY}"/*.zip "$SITE_HOME/assets/"
   rm -rf "$TLDR_LANG_ARCHIVES_DIRECTORY"
   cp -f "$TLDRHOME/index.json" "$SITE_HOME/assets/"
-  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages.pdf" "${SITE_HOME}/assets/tldr-book.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-book*.pdf" "${SITE_HOME}/assets/"
+
 
   sha256sum \
     "${SITE_HOME}/assets/index.json" \
-    "${SITE_HOME}/assets/"*.zip \
```

Comment:
This line should not have been deleted.

The command here should be:
```
  sha256sum \
    "${SITE_HOME}/assets/index.json" \
    "${SITE_HOME}/assets/"*.zip \
    "${SITE_HOME}/assets/tldr-book*.pdf" \
    > "${SITE_HOME}/assets/tldr.sha256sums"
```

Reply:
Nice catch, VSCode for some reason removed the line while I applied your suggestion. Just a minute, will fix it.